iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/RESTService.java

```diff
@@ -110,6 +112,9 @@ private void initServer(IcebergConfig icebergConfig) {
         new AbstractBinder() {
           @Override
           protected void configure() {
+            bind(IcebergRESTAuthInterceptionService.class)
```

Comment:
Could we not register InterceptionService like https://github.com/apache/gravitino/pull/7566?